dawn/src/dawn/Compiler/Driver.cpp

```diff
@@ -0,0 +1,64 @@
+//===--------------------------------------------------------------------------------*- C++ -*-===//
+//                          _
+//                         | |
+//                       __| | __ ___      ___ ___
+//                      / _` |/ _` \ \ /\ / / '_  |
+//                     | (_| | (_| |\ V  V /| | | |
+//                      \__,_|\__,_| \_/\_/ |_| |_| - Compiler Toolchain
+//
+//
+//  This file is distributed under the MIT License (MIT).
+//  See LICENSE.txt for details.
+//
+//===------------------------------------------------------------------------------------------===//
+
+#include "dawn/Compiler/Driver.h"
+#include "dawn/CodeGen/TranslationUnit.h"
+#include "dawn/Compiler/DawnCompiler.h"
+#include "dawn/Compiler/Options.h"
+
+namespace dawn {
+
+std::list<PassGroup> defaultPassGroups() {
```

Comment:
should this live here? Would this not be nicer in the optimizer?

Reply:
The next PR does this - #886. In this PR it should be in Optimizer/OptimizerOptions, but that doesn't have a corresponding source file yet.